src/snowflake/snowpark/modin/pandas/new_types_demo.py

```diff
@@ -0,0 +1,52 @@
+import modin.pandas as pd
+import snowflake.snowpark.modin.plugin
+import numpy as np
+import pandas as native_pd
+from snowflake.snowpark.session import Session; session = Session.builder.create()
+import logging; logging.getLogger("snowflake.snowpark").setLevel(logging.DEBUG)
+
```

Comment:
for demo purposes would recommend a jupyter noteboook, just easier as you can quickly modify code.